hack/import-restrictions.yaml

```diff
@@ -142,23 +149,29 @@
 - baseImportPath: "./vendor/k8s.io/kube-controller-manager/"
   allowedImports:
   - k8s.io/apimachinery
-  - k8s.io/apiserver
+  - k8s.io/component-base
+  - k8s.io/klog
   - k8s.io/utils
 
 - baseImportPath: "./vendor/k8s.io/kube-proxy/"
   allowedImports:
   - k8s.io/apimachinery
+  - k8s.io/component-base
+  - k8s.io/klog
+  - k8s.io/utils
 
 - baseImportPath: "./vendor/k8s.io/kube-scheduler/"
   allowedImports:
   - k8s.io/apimachinery
-  - k8s.io/apiserver
+  - k8s.io/component-base
+  - k8s.io/klog
```

Comment:
After using `k8s.io/component-base`, can we drop both  `k8s.io/apimachinery` and `k8s.io/apiserver` in `allowedImports` list? 
and in others place?
may be a big work, should be a follow-up PR

Reply:
afaik there is no transitive closure computation. Feel free to add it.